apps/src/templates/sectionProgress/sectionProgressTestHelpers.js

```diff
@@ -92,7 +92,7 @@ function buildSectionProgress(students, scriptData) {
     studentLessonProgressByUnit: {
       [scriptData.id]: lessonProgressForSection(progress, scriptData.lessons),
     },
-    studentLastUpdateByUnit: {[scriptData.id]: lastUpdates},
+    studentLastUpdateByUnit: lastUpdates,
```

Comment:
this seems to have been just completely broken; it was double-nesting as `{scriptData.id: {scriptData.id: lastUpdates}}`